FishContainment/schema.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "http://json-schema.org/schema#",
```

Comment:
The definitions of the spec.md have to be included in this file. Please see contribution manual pages 8 and 9.
[https://bit.ly/contribution_manual](https://bit.ly/contribution_manual)